File: VB/Scaffolding.ValidationErrors/Common/Utils/DbExtensions.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections
Imports System.Linq

Namespace Scaffolding.ValidationErrors.Common.Utils
	Public Module DbExtensions
        <System.Runtime.CompilerServices.Extension> _
        Public Sub Load(ByVal source As IQueryable)
            Dim enumerator As IEnumerator = source.GetEnumerator()
            Do While enumerator.MoveNext()
            Loop
        End Sub
	End Module
End Namespace

File: CS/Scaffolding.ValidationErrors/Common/Utils/DbExtensions.cs
using System;
using System.Collections;
using System.Linq;

namespace Scaffolding.ValidationErrors.Common.Utils {
    public static class DbExtensions {
        public static void Load(this IQueryable source) {
            IEnumerator enumerator = source.GetEnumerator();
            while(enumerator.MoveNext()) {
            }
        }
    }
}
